.gitmodules

```diff
@@ -1,6 +1,7 @@
 [submodule "lib/pkp"]
 	path = lib/pkp
 	url = git@github.com:pkp/pkp-lib.git
+	branch = ojs-alpha_3_0
```

Comment:
I guess this means you're working with the 3.0a branch? That's a very old branch -- the master branch is currently both more advanced and more stable. If you'd like to experiment I'd recommend checking it out :)